### PR TITLE
Better fix for the IntegerStateQuery blendfunc test

### PR DESCRIPTION
**Pull request by Kangz**
Make a better version of the fix for the IntegerStateQuery as suggested in https://github.com/KhronosGroup/WebGL/pull/1685

### CODE REVIEW

File: sdk/tests/conformance2/state/gl-initial-state.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+<canvas id="canvas" width="2" height="2"> </canvas>
+<script>
+"use strict";
+description("This test checks the initial value of the OpenGL state");
+
+debug("");
+debug("Canvas.getContext");
+
+var wtu = WebGLTestUtils;
+var gl = wtu.create3DContext("canvas", null, 2);
```
Comment (zhenyao) [inline sdk/tests/conformance2/state/gl-initial-state.html]: Sorry to mislead you, but can you move this test to conformance/state/ and leave this create3DContext without a version parameter, so we could run it in both WebGL1 and WebGL 2? For the initial settings in this test, they are also available in WebGL1.

Comment (Kangz) [inline sdk/tests/conformance2/state/gl-initial-state.html]: Done

File: sdk/tests/conformance/state/00_test_list.txt [hunk 2]
```diff
@@ -5,4 +5,4 @@ gl-geterror.html
 --max-version 1.9.9 gl-getstring.html
 --max-version 1.9.9 gl-object-get-calls.html
 --min-version 1.0.3 state-uneffected-after-compositing.html
-
+gl-initial-state.html
```
Comment (kenrussell) [inline sdk/tests/conformance/state/00_test_list.txt]: Please alphabetize and add --min-version 1.0.4. Thanks.

Comment (Kangz) [inline sdk/tests/conformance/state/00_test_list.txt]: Done

**Comment by kenrussell** [comment]
LGTM. Thanks. Let's let @zhenyao re-review and merge.

**Comment by zhenyao** [comment]
lgtm.

**Comment by Kangz** [comment]
Thank you for the merge, please tell me when this is rolled so I can remove the expectation.

**Pull request merged**